src/utils/utils.cpp

```diff
@@ -409,6 +597,59 @@ void EchoInfo(const std::string &message) {
   }
 }
 
+void EchoStats(const std::map<std::string, std::int64_t> &stats) {
+  for (const auto &[key, value] : stats) {
+    if (value == 0) {
+      continue;
+    } else if (key == "nodes-created") {
+      std::cout << value << " nodes have been created." << std::endl;
+    } else if (key == "nodes-deleted") {
+      std::cout << value << " nodes have been deleted." << std::endl;
+    } else if (key == "relationships-created") {
+      std::cout << value << " relationships have been created." << std::endl;
+    } else if (key == "relationships-deleted") {
+      std::cout << value << " relationships have been deleted." << std::endl;
+    } else if (key == "labels-added") {
+      std::cout << value << " labels have been created." << std::endl;
+    } else if (key == "labels-removed") {
+      std::cout << value << " labels have been deleted." << std::endl;
+    } else if (key == "properties-set") {
+      std::cout << value << " properties have been updated." << std::endl;
+    }
+  }
+}
+
+void EchoNotification(const std::map<std::string, std::string> &notification) {
+  std::string severity = notification.at("severity");
+  std::transform(severity.begin(), severity.end(), severity.begin(), ::toupper);
+  if (severity == "WARNING") {
```

Comment:
Consider doing this in `ParseNotifications`, because in the future we might need to check the severity in other places. Also as we don't have to support other databases, we might "hardcode" the structure of our notifications and have a proper structure for them where the severity can be an enum.

P.S.: we should have some common code between mgconsole and Memgraph, because the notification is a very good example for something that we are using in both projects. But this is an another topic.